static/app/utils/profiling/flamegraph/flamegraphTheme.tsx

```diff
@@ -105,28 +103,33 @@ export const LCH_DARK = {
   L_d: 0.1,
 };
 
+const SIZES: FlamegraphTheme['SIZES'] = {
+  BAR_FONT_SIZE: 11,
+  BAR_HEIGHT: 20,
+  BAR_PADDING: 4,
+  FLAMEGRAPH_DEPTH_OFFSET: 12,
+  FOCUSED_FRAME_BORDER_WIDTH: 2,
+  FRAME_BORDER_WIDTH: 2,
+  GRID_LINE_WIDTH: 2,
+  HOVERED_FRAME_BORDER_WIDTH: 1,
+  INTERNAL_SAMPLE_TICK_LINE_WIDTH: 1,
+  LABEL_FONT_PADDING: 6,
+  LABEL_FONT_SIZE: 10,
+  MINIMAP_HEIGHT: 100,
+  MINIMAP_POSITION_OVERLAY_BORDER_WIDTH: 2,
+  SPANS_HEIGHT: 100,
+  SPANS_BAR_HEIGHT: 16,
+  TIMELINE_HEIGHT: 20,
+  TOOLTIP_FONT_SIZE: 12,
+};
+
+const FONTS: FlamegraphTheme['FONTS'] = {
+  FONT: MONOSPACE_FONT,
+  FRAME_FONT,
+};
+
 export const LightFlamegraphTheme: FlamegraphTheme = {
-  CONFIG: {
-    HIGHLIGHT_RECURSION: false,
-  },
-  SIZES: {
```

Comment:
Sizes and fonts were all just duplicated between light/dark theme and they actually need to be kept in sync, so centralized it.